src/Symfony/Component/Security/Http/Authentication/DefaultAuthenticationSuccessHandler.php

```diff
@@ -84,10 +84,8 @@ public function getProviderKey()
 
     /**
      * Set the provider key.
```

Comment:
I think this entire doc block can be removed now, as the comment doesn't tell us anything anymore